src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features_prop.json

```diff
@@ -1,4 +1,4 @@
-#Wed Jul 22 11:25:50 Pacific Daylight Time 2020
+#Fri Jul 24 17:56:48 China Standard Time 2020
```

Comment:
Does this comments affect test result?

Reply:
This is automatically generated and should not affect the test results